test/fixtures/05-use-file-name/before.js

```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components";
 
 const Test = styled.div`color: red;`;
+const before = styled.div`color: blue;`;
```

Comment:
I'm not sure I follow exactly. Isn't that what `Test` is directly above this component?